app/assets/stylesheets/application.scss

```
@@ -0,0 +1,2 @@
+@import 'bootstrap-sprockets';
+@import 'bootstrap';
```

You need to require files in the folder app/assets/stylesheets
```suggestion
/*
*= require_tree .
*= require_self
*/

@import 'bootstrap-sprockets';
@import 'bootstrap';
```